src/components/App.js

```diff
@@ -27,6 +29,9 @@ import useTaxonCommonNames from "./hooks/useTaxonCommonNames";
 
 const { useRealm } = RealmContext;
 
+// 200MB - number in bytes
+const MIN_DEVICE_STORAGE = 200000000;
```

Comment:
IMO, long numbers are a lot easier to read with separators, e.g. `200_000_000`. It could also use a name that better describes its purpose. Is this referring to total size of the disk? The smallest amount of *available* disk we can use?